### ISSUE TITLE
BCE threshold dropping during eval

### ISSUE THREAD

**Issue by brianhance** [UKPLab/sentence-transformers#1391]
Hi, when fine tuning, I have noticed that during training, acc threshold for BCE decreases substantially while acc either maintains or slightly increases. Does this mean the model is overfitting, or is becoming biased? I am training with MNR, and have exactly the same number of pos and negs in my test set. Thanks

**Comment by nreimers** [comment]
What is BCE? BinaryClassificationEvaluator?

Yes, it is normal: When you train longer, clusters / classes get better separated, hence a lower threshold is possible.

**Comment by brianhance** [comment]
Thanks - that makes sense. Would you say a decreasing threshold is an indicator of the model improving, or should i only refer to acc, f1, etc.

**Comment by nreimers** [comment]
Better to refer to F1 etc. The best measure to look at is in my opinion is Average Precision (AP).

**Comment by brianhance** [comment]
sounds good - thanks.
